api/queries_issue.go

```diff
@@ -33,6 +30,7 @@ type Issue struct {
 	Body           string
 	CreatedAt      time.Time
 	UpdatedAt      time.Time
+	ClosedAt       *time.Time
```

Comment:
The other time fields are guaranteed to be present (non-null), but this one isn't, and by marking it a pointer we grant it a possible null state. This matter for the exported JSON, because otherwise a zero-value timestamp would get exported.